src/scanoss/file_filters.py

```diff
@@ -232,14 +236,18 @@ class FileFilters(ScanossBase):
     """
 
     def __init__(
-        self,
-        debug: bool = False,
-        trace: bool = False,
-        quiet: bool = False,
-        scanoss_settings: 'ScanossSettings | None' = None,
-        all_extensions: bool = False,
-        all_folders: bool = False,
-        hidden_files_folders: bool = False,
+            self,
+            debug: bool = False,
+            trace: bool = False,
+            quiet: bool = False,
+            scanoss_settings: 'ScanossSettings | None' = None,
+            all_extensions: bool = False,
+            all_folders: bool = False,
+            hidden_files_folders: bool = False,
+            operation_type: str = 'scanning',
+            skip_size: int = 0,
+            skip_extensions = None,
+            skip_folders = None
```

Comment:
i would add the types here for skip options

also i'm not 100% convinced we need to pass them to the constructor.. it's a little confusing with the other arguments (`all_extensions, all_folders, hidden_files_folders`)

maybe we can create setters for these, because we could want to add them after instantiation?

Reply:
Also, are we sure we want to set operation type as default? from my pov this should be a required argument then